Deploy-CAM.ps1

```diff
@@ -1201,7 +1280,7 @@ function Deploy-CAM()
 	Write-Host "Using temporary directory $tempDir for intermediate files"
 	if(-not (Test-Path $tempDir))
 	{
-		New-Item $tempDir -type directory > $null
+		New-Item $tempDir -type directory | Out-Null
```

Comment:
It definitely doesn't matter in this case but I found this posting interesting re. performance of the various 'don't pollute the pipeline' strategies :https://stackoverflow.com/questions/5260125/whats-the-better-cleaner-way-to-ignore-output-in-powershell

Reply:
Using pipeline Out-Null purpose is to wait for the cmdlet complete finish. If this is not the case, then do not using pipepline Out-Null here. That is why I use Out-Null instead of $null.

Reply:
Okay. Cool. I wasn't thinking about waiting.